dash/backend/src/modules/falco/controllers/falco.controller.ts

```diff
@@ -154,33 +160,27 @@ export class FalcoController {
         @Query('key') key: string
     ): Promise<FalcoDto> {
         // look up user by api key
-        const currentUserAuthObj = await this.userDao.loadUserByApiKey(key);
+        const currentUserAuthObj = await this.userDao.loadUserByApiKey(key || '');
         if (!currentUserAuthObj) {
             throw new UnauthorizedException('Access Denied!');
         }
 
         // get all authorities from the current user
         const currentUserAuth = currentUserAuthObj[0].authorities;
-        let isFalcoUser = this.authService.checkAuthority(currentUserAuth, [AuthorityId.FALCO]);
+        const isFalcoUser = this.authService.checkAuthority(currentUserAuth, [AuthorityId.FALCO]);
         if (!isFalcoUser) {
             throw new UnauthorizedException('Access Denied!');
         }
 
-        // @TODO: Will implement falco filters/rules settings at a later time
-        // check if falco settings rules should be applied
-        /*
-            const rules = getFalcoSettingsRules(clusterId);
-            for (rules : rule) {
-                if (rule.isRelevant(event)) {
-                    if (rule.affect === 'ignore') {
-                        return; // don't save this event - it is being filtered out
-                    }
-                }
-            }
-        */
-
-        // Saved new falco log
-        return await this.falcoService.createFalcoLog(clusterId, falcoLog);
+        const action = await this.falcoService.checkRules(clusterId, falcoLog);
+        // An ignore rule applied,
+        if (action === FalcoRuleAction.Ignore) {
+            this.logger.log('Ignoring Falco Event');
```

Comment:
Maybe should include some details about the event, otherwise you'll just know that "some event" was ignored but nothing about which event

Reply:
Added in latest commit, now includes cluster ID, namespace, image, and rule name